Can't deploy Desktop project with Gradle.

#### Issue details
I am using the 1.9.10 libGDX version with the Libgdx Project Generator. I create a new project with only Desktop selected and I generate it. The generation is successful, so I continue by opening the project in IntelliJ IDEA 2019.1.3 Community Edition (Windows 10 Home 64 bit). Even though the default red background with the libGDX logo application runs fine, I just can't deploy it with Gradle to Desktop. I am using JDK 1.8.0_211 but the same happens with JDK 12.0.1

#### Reproduction steps/code
Create a new project with the Libgdx Project Generator only for Desktop and with no extensions, open it in IntelliJ IDEA, go to Terminal tab at the bottom and type gradlew desktop:dist.

#### Version of LibGDX and/or relevant dependencies
1.9.10 with no extensions as per Libgdx Project Generator

#### Stacktrace
```java
> Task :core:compileJava FAILED
error: Source option 6 is no longer supported. Use 7 or later.
error: Target option 6 is no longer supported. Use 7 or later.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:compileJava'.
> Compilation failed; see the compiler error output for details. 
```

After getting that error I changed the build.gradle in Core and Desktop sourceCompatibility to 1.8. But I still get an error:
```java
> Task :desktop:dist FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':desktop:dist'.
> Failed to create MD5 hash for file '...\Deploy\core\build\libs\core-1.0.jar' as it does not exist.
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [x] Windows
- [ ] Linux
- [ ] MacOS

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/desktop/build.gradle
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/desktop/build.gradle
@@ -28,8 +28,9 @@ task dist(type: Jar) {
     manifest {
         attributes 'Main-Class': project.mainClassName
     }
+    dependsOn configurations.runtimeClasspath
     from {
-        configurations.compileClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
     }
     with jar
 }

